packages/expo-device/src/Device.ts

```diff
@@ -54,27 +259,86 @@ export async function getMaxMemoryAsync(): Promise<number> {
   return maxMemory;
 }
 
+/**
+ * > **warning** This method is experimental and is not completely reliable. See description below.
+ *
+ * Checks whether the device has been rooted (Android) or jailbroken (iOS). This is not completely reliable because there exist solutions to bypass root-detection
+ * on both [iOS](https://www.theiphonewiki.com/wiki/XCon) and [Android](https://tweakerlinks.com/how-to-bypass-apps-root-detection-in-android-device/).
+ * Further, many root-detection checks can be bypassed via reverse engineering.
+ * - On Android, it's implemented in a way to find all possible files paths that contain the `"su"` executable but some devices that are not rooted may also have this executable. Therefore, there's no guarantee that this method will always return correctly.
+ * - On iOS, [these jailbreak checks](https://www.theiphonewiki.com/wiki/Bypassing_Jailbreak_Detection) are used to detect if a device is rooted/jailbroken. However, since there are closed-sourced solutions such as [xCon](https://www.theiphonewiki.com/wiki/XCon) that aim to hook every known method and function responsible for informing an application of a jailbroken device, this method may not reliably detect devices that have xCon or similar packages installed.
+ * - On web, this always resolves to `false` even if the device is rooted.
+ * @return Returns a promise that resolves to a `boolean` that specifies whether this device is rooted.
+ * @example
+ * ```js
+ * await Device.isRootedExperimentalAsync();
+ * // true or false
+ * ```
+ */
 export async function isRootedExperimentalAsync(): Promise<boolean> {
   if (!ExpoDevice.isRootedExperimentalAsync) {
     throw new UnavailabilityError('expo-device', 'isRootedExperimentalAsync');
   }
   return await ExpoDevice.isRootedExperimentalAsync();
 }
 
+/**
+ * **Using this method requires you to [add the `REQUEST_INSTALL_PACKAGES` permission](/config/app#permissions).**
+ * Returns whether applications can be installed for this user via the system's [Intent#ACTION_INSTALL_PACKAGE](https://developer.android.com/reference/android/content/Intent.html#ACTION_INSTALL_PACKAGE)
+ * mechanism rather than through the OS's default app store, like Google Play.
+ * @return Returns a promise that resolves to a `boolean` that represents whether the calling package is allowed to request package installation.
+ * @example
+ * ```js
+ * await Device.isSideLoadingEnabledAsync();
+ * // true or false
+ * ```
+ * @platform android
+ */
 export async function isSideLoadingEnabledAsync(): Promise<boolean> {
   if (!ExpoDevice.isSideLoadingEnabledAsync) {
     throw new UnavailabilityError('expo-device', 'isSideLoadingEnabledAsync');
   }
   return await ExpoDevice.isSideLoadingEnabledAsync();
 }
 
+/**
+ * Gets a list of features that are available on the system. The feature names are platform-specific.
+ * See [Android documentation](<https://developer.android.com/reference/android/content/pm/PackageManager#getSystemAvailableFeatures()>)
```

Comment:
Make sure this link gets rendered correctly

Reply:
Link is working correctly, it's wrapped with `<>` because link contains brackets in URL. It's one of possible workarounds for that case.

<img width="624" alt="Screenshot 2022-11-29 at 09 07 29" src="https://user-images.githubusercontent.com/719641/204474752-4ee987ba-659d-4dc0-9716-960ace009f6d.png">